.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+indent_size = tab
+tab_width = 4
+
+[{*.json,*.yml,*.md}]
```

Comment:
@f3l1x Could you, please, remove json from this list? Thanks a lot!

Reply:
Done